src/core/src/package_managers/ZypperPackageManager.py

```diff
@@ -156,6 +166,17 @@ def invoke_package_manager(self, command):
                 self.force_reboot = True
             return out
 
+    def modify_upgrade_or_patch_cmd_to_replacefiles(self, cmd):
```

Comment:
Minor: use the same variable names for the same entity even if it doesn't matter in a programmatic call stack: command instead of cmd here.